CHANGELOG.md

```diff
@@ -4,67 +4,67 @@
 
 **Implemented New Features and Enhancements**
 
--   Update PrimeVue version
+- Update PrimeVue version
```

Comment:
As this file was small, I let prettier fix, but I'm happy to revert this and add it to the `.prettierignore` if you like?